cmake/linker/ld/target_relocation.cmake

```diff
@@ -3,28 +3,31 @@
 # See root CMakeLists.txt for description and expectations of these macros
 
 macro(toolchain_ld_relocation)
-  set(MEM_RELOCATAION_LD   "${PROJECT_BINARY_DIR}/include/generated/linker_relocate.ld")
-  set(MEM_RELOCATAION_SRAM_DATA_LD
+  set(MEM_RELOCATION_LD   "${PROJECT_BINARY_DIR}/include/generated/linker_relocate.ld")
```

Comment:
Did you make some whitespace changes or something? I can't tell the difference between the before and after, it looks like the code is unchanged.

Reply:
Disregard, I see it now